centrallix-sysdoc/OSDriver_Authoring.md

```diff
@@ -115,497 +170,1032 @@ License:  Copyright (C) 2001-2011 LightSys Technology Services. See LICENSE.txt
     - [B.	Object attribute enumeration, getting, and setting.](#bobject-attribute-enumeration-getting-and-setting)
     - [C.	Object querying (for subobjects)](#cobject-querying-for-subobjects)
 
+
+
 ## I Introduction
-An objectsystem driver's purpose is to provide access to a particular type of local or network data/resource, and to organize that data in a tree- structured heirarchy that can be integrated into the Centrallix's ObjectSystem.  This tree structure will vary based on the data being presented, but will fit the basic ObjectSystem model of a heirarchy of objects, each having attributes, perhaps some methods, and possibly content.  
+An objectsystem driver's purpose is to provide access to a particular type of local or network data/resource.  Specific information about the resource to be accessed (such as credentials for a database, queries for selecting data, the auth token for an API, etc.) is stored in a file that is openned by the relevant driver.  For example, the query driver (defined in `objdrv_query.c`) opens `.qy` files, which store one or more ObjectSQL queries used to fetch data.
```

Comment:
openned -> opened (and in other locations)

Reply:
I cannot figure out why my spell check doesn't catch that. I'll keep an eye out for it in the future.

Fixed.